### ISSUE TITLE
Cloud: Labour Edit - Customer Quoted hours become Mechanic Estimated Hours

### ISSUE THREAD

**Issue by giovanicani** [GCSO2/D3S-QUOTE#153]
As per discussion this morning with development team.

Issue was also shown on screen share - skype.

Waiting on the fix to approve move to prod.

**Comment by giovanicani** [comment]
works fine now... however another issue found will create a new ticket.
